beetsplug/vocadb.py

```diff
@@ -1,12 +1,18 @@
+from collections.abc import Iterable, Sequence
 from datetime import datetime
 from itertools import chain
 from json import load
+from optparse import Values
 from re import Match, match, search
-from typing import Any, NamedTuple, Optional, Sequence
+from typing import NamedTuple, TypedDict, TYPE_CHECKING
```

Comment:
```suggestion
from typing import NamedTuple, TypedDict, TYPE_CHECKING, Union
```
Python 3.9 doesn't support "X | Y", and needs Union